Explorer/Assets/DCL/Navmap/FloatingPanelView.cs

```diff
@@ -99,5 +100,14 @@ public class FloatingPanelView : MonoBehaviour
         [field: Header("Audio")]
         [field: SerializeField]
         public AudioClipConfig OnShowAudio { get; private set; }
+
+        internal UnityAction? onPointerEnterAction;

```

Comment:
I'd prefer the usage of generic C# system actions instead of the Unity re-inplementation